### PR TITLE
[#407] [FEATURE] Affichage d'une médaille avec le nombre de pixs gagnés si l'utilisateur n'obtient pas de trophée dans un test adaptatifs (US-473)

### PR DESCRIPTION
**Pull request by MBrandone**

### CODE REVIEW

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

File: live/app/components/scoring-panel.js [hunk 1]
```diff
@@ -2,8 +2,10 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
 
-  assessment: null,
+  classNames : ['scoring-panel'],
```
Comment (florianEnoh) [inline live/app/components/scoring-panel.js:5]: classNames: ['scoring-panel'] //Le linter va te taper sur les doigts - pas d'espace avant les :

File: live/app/components/medal-item.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  classNames : ['medal-item'],
+
+  pixScore: null
```
Comment (florianEnoh) [inline live/app/components/medal-item.js:6]: Si c'est une variable privée, il est conseillé de préfixer avec _ (_pixScore)

**Review by florianEnoh — COMMENTED** [review]
LGTM modulo les changements.

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Comment by jbuget** [comment]
I've deployed this PR to http://473-scoring-panel-won-pix.pix.beta.gouv.fr. Please check it out

**Pull request merged**